src/main/java/it/pagopa/pn/client/b2b/pa/PnPaB2bUtils.java

```diff
@@ -177,7 +177,7 @@ public  it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model
 
     public NewNotificationResponse uploadNotification50size( NewNotificationRequestV21 request) throws IOException {
```

Comment:
il metodo non ha un nome corrispondende al file che viene caricato, in generale comunque sarebbe meglio non implementare un nuovo metodo per cambiare solamente il file da caricare ma generalizzare il metodo di upload e passare diversi file, un esempio banale, si può fare decisamente meglio, ma così è abbastanza rapido sistemare sarebbe:

 public NewNotificationResponse uploadNotification50size( NewNotificationRequestV21 request) throws IOException {
 return uploadNotification(request,"classpath:/AllegatoServiceDesk_100pag.pdf");
 }
 
  private NewNotificationResponse uploadNotification( NewNotificationRequestV21 request, String fileToUpload) throws IOException {
  //codice per upload tra cui 
  // NotificationDocument notificationDocument = newDocument(fileToUpload);
  }

Reply:
modificato ho usato il caricamento degli allegati con tot pagine introdotto con i 20 grammi siccome il pdf era un normale pdf con 100 pagine